### ISSUE TITLE
Env CPU Rendering

### ISSUE THREAD

**Issue by XMaster96** [iksnagreb/gym-csgo#1]
Hello, looks like a cool project.
Is it possible to fully render the env on CPU only?

thanks

**Comment by iksnagreb** [comment]
Yes, I think this should be possible. It is pretty much what is happening if you run the game on the virtual display without enabling Vulkan or compatibility mode (Proton). The performance will probably be quite bad, though - you should expect no more than ~ 10 fps. However, I have not tested so far, whether it is necessary to have at least a GPU installed to start.

If you have any concrete problems or suggestions, please let me know.

**Comment by Okonai** [comment]
@iksnagreb 
I'm not sure you still active on this repo, but I've been learning python with chatGPT lately to create an AI that would be trained to play Guilty Gear Strive. I came up with a solution using GeforceNow Ultimate membership, and DXcam library to capture up to 240 fps screen grabs and running python up to 3.9 ( this is the highest pytorch supported version, gymnasium supports up to 3.11) on windows using CUDA. 

This way I use little to no resources to run the game, around 400MB of ram and not much CPU so my GPU can fully be leveraged by the training. 

Point is, let me know, if you are open to mentor me a bit and make this environment better as well as the one for GGST as well, cause I don't know jack**** about AI and not much python either, yet. 

Best regards
